readme.md

```diff
@@ -311,6 +311,23 @@ You can also use your own local Babel version:
 }
 ```
 
+*Note:  AVA currently only transpiles the tests that you ask it to run.  In other words, it won't transitively transpile modules that you ```import``` from outside of the test (that also may happen to be written in ES6 and/or use the ES6 module syntax).  While there are valid reasons for taking this approach (no impact/modification to the code under test), it may not be what you expect!*
+
+*As a simple workaround, you can use [Babel's require hook](https://babeljs.io/docs/usage/require/) in order to do on-the-fly transpiling of modules that are subsequently required.  Because AVA supports ES6 module syntax, you can use it to import the require hook itself:*
+
+```js
+import 'babel-core/register';
```

Comment:
If you move it down it will not have an effect on the below import which is AVA. AVA is pretty large, so it will definitely slow it down if it needs to be transpiled too.